src/agentscope/models/dashscope_model.py

```diff
@@ -214,6 +215,107 @@ def format(
     ) -> List:
         """Format the messages for DashScope Chat API.
 
+        In this format function, the input messages are converted into
+        dictionaries with `role` and `content` fields. This conversation may
+        not meet the requirement that `user` and `assistant` speak
+        alternatively. It'll be correct in `preprocess_role` function.
```

Comment:
"it'll be correct" -> "This requirement can be enforced by calling `preprocess_role` function."

Reply:
Modified accordingly